core/src/main/java/org/htsjdk/core/api/PathURI.java

```diff
@@ -0,0 +1,86 @@
+package org.htsjdk.core.api;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URI;
+import java.nio.file.Path;
+
+/**
+ * Interface representing htsjdk-next input/output resources as URIs.
+ */
+public interface PathURI {
+
+    /**
+     * @return true if this URI has a scheme that has an installed NIO file system provider. This does not
+     * guarantee the URI can be converted into a {@code java.nio.file.Path}, since the URI can be syntactically
+     * valid, and specify a valid file system provider, but still fail to be semantically meaningful.
```

Comment:
I prefer to have properly formatted javadocs. Most of the information should be at the begining, and the `@return` tag should contain just the information about the return value (in this case, `{@code true} if the URI has a installed NIO filesystem provider; {@code false} otherwise`).

Reply:
Done.